### PR TITLE
Bug for default locales

### PR DESCRIPTION
**Pull request by jspeis**
Here is a proposed fix for handling captioning of labels with default languages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,11 +50,12 @@ Note that a systemd `.service` is also installed. You will probably need to modi
 
 ### Environment Variables
 - `TESSERACT_DATABASE_URL`: required, is the address of the database; make sure to include the user, password, and database name.
-- `TESSERACT_SCHEMA_FILEPATH`: required, should point to the location on disk for the tesseract schema file.
-- `TESSERACT_LOGIC_LAYER_CONFIG_FILEPATH`: optional, should point to the location on path for the logic layer configuration.
+- `TESSERACT_DEBUG`: boolean, `true` is a debug mode for data/calculations. As of now, the only feature is the `rca` calculation will show `a b c d` sub-aggregations for the calculation`(a/b)/(c/d)`.
```
Comment (MarcioPorto) [inline README.md]: I think while we're at it we should update this text. I believe now setting this to true also returns mote detailed error messages in the response when queries fail, so it goes beyond just RCA.

**Comment by jspeis** [comment]
@hwchen let me know if this looks good and then we can merge

File: tesseract-core/src/schema.rs [hunk 2]
```diff
@@ -138,10 +139,13 @@ impl From<SchemaConfigJson> for Schema {
                     .collect()
             });
 
+        let default_locale_str: &str = "en";
```
Comment (hwchen) [inline tesseract-core/src/schema.rs]: can we make this a `const DEFAULT_LOCALE_STR` and put it at the top of the schema? (that's my preference, anyways, let me know if you prefer this way of handling hardcoded stuff)

Comment (jspeis) [reply tesseract-core/src/schema.rs]: @hwchen agreed! I've just updated this

**Comment by hwchen** [comment]
Thanks!

**Pull request merged**